src/libmongoc/src/mongoc/mongoc-client.c

```diff
@@ -306,7 +306,6 @@ _mongoc_hstrerror (int code)
    case HOST_NOT_FOUND:
       return "The specified host is unknown.";
    case NO_ADDRESS:
-   case NO_DATA:
```

Comment:
Is this guaranteed to have the same value as `NO_ADDRESS`? I assume you caught this in some CI warning?

Reply:
Yes. I added both constants to ensure we're grabbing it, but CI complained about the case being duplicated. I'd argue that using one of these should be sufficient, and the 50:50 joker picked `NO_DATA` for elimination. I don't anticipate trouble from this, but I suppose a last resort to deal with either constant not being defined would be to hardcode the value (`4`) and be done with it.